src/test/java/WebDriverSeleniumTests/GoogleSearchForWikiEdge.java

```diff
@@ -0,0 +1,91 @@
+package WebDriverSeleniumTests;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class GoogleSearchForWikiEdge {
+
+    @Test
+    public void GoogleSearchForWikiIsNotEmpty() throws IOException {
```

Comment:
- methods starts from small letter
- your test name didn't describe the purpose of test, at least add browser name into name (e.g. `...ForEdge` )

Reply:
Ups)